### PR TITLE
[linker/ru] Add description of linker script

### PR DESCRIPTION
**Pull request by Zamuhrishka**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: ru-ru/linker-ru.html.markdown [hunk 1]
```diff
@@ -0,0 +1,204 @@
+---
+category: tool
+tool: linker
+contributors:
+    - ["Alexander Kovalchuk", "https://github.com/Zamuhrishka"]
+translators:
+
```
Comment (divayprakash) [inline ru-ru/linker-ru.html.markdown]: Please change this to

```
contributors:
    - ["Alexander Kovalchuk", "https://github.com/Zamuhrishka"]
translators:
    - ["Alexander Kovalchuk", "https://github.com/Zamuhrishka"]
```

and also remove the ```filename:``` line

**Comment by divayprakash** [comment]
@adambard Hey! Need a little help with this one. This is a new article but in Russian. I am able to get it to build once the translators name is put in however it doesn't show up on the homepage. Manually accessing ```/docs/ru-ru/linker-ru/``` works and page is built successfully. Could you take a look?

File: ru-ru/linker-ru.html.markdown (hunk 1 above)
Comment (Zamuhrishka) [reply ru-ru/linker-ru.html.markdown]: Thank you. I made changes which you asked.

**Comment by divayprakash** [comment]
@Zamuhrishka It'll be a little while before I can merge this due to some errors with the build. I'll keep you posted.

**Comment by adambard** [comment]
Thanks for your contribution! I had to create an english-language stub so that the site knew were to display the russian version, it should be live shortly.

**Comment by Zamuhrishka** [comment]
Thanks for all. :)

**Pull request merged**